2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,11 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let report=[]
+    for(let i = 0; i < cities.length; i++){
+report[i]=`The temperature in ${cities[i]} is ${temperatureService(cities[i])} degrees`;
```

Comment:
nice use of template literals! here you can use push, for example `report.push(`The temperature in ${cities[i]} is ${temperatureService(cities[i])} degrees`)`